src/details/ArborX_DetailsHappyTreeFriends.hpp

```diff
@@ -49,6 +51,51 @@ struct HappyTreeFriends
     return bvh.getBoundingVolume(node);
```

Comment:
Check if we can remove this function in the PR. Maybe we can also remove `getRoot()`, and introduce `getRootIndex()`. So far, the only case when root index may not be zero is when constructing a hierarchy based on Apetrei indices.